### PR TITLE
minor refactor to remove log error when thowing exceptions

### PR DESCRIPTION
**Pull request by iguannago**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java [hunk 1]
```diff
@@ -57,8 +57,8 @@ public void process() {
             log.debug("*** case-loader *** file validated successfully: {}", file.getName());
             if (file.isDelta()) {
                 if (null == latestRef) {
-                    String message = String.format("No reference data processed for this delta: %s", file.getName());
-                    throw new TransformException(message);
+                    throw new TransformException(String.format("No reference data processed for this delta: %s",
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java]: Do you need to log this error?

Comment (iguannago) [reply src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java]: Yes, it's good to know what delta file is failing.

File: build.gradle [hunk 2]
```diff
@@ -25,7 +25,7 @@ compileTestJava {
 
 targetCompatibility = 1.8
 
-mainClassName = 'uk.gov.hmcts.reform.sscs.CaseLoaderApp'
+mainClassName = 'uk.gov.hmcts.sscs.CaseLoaderApp'
```
Comment (jackmaloney) [inline build.gradle]: Doesn't the package name of `CaseLoaderApp` have reform in it?

File: src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java (hunk 1 above)
Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/services/CaseLoaderService.java]: Still no logging so this won't show on app insights - you need a `LOG.error(message, exception);` Reference Notification Service

**Pull request merged**